ecommers/lib/ui/widgets/authorization/auth_text_field.dart

```diff
@@ -6,21 +6,24 @@ import 'package:flutter_svg/svg.dart';
 class AuthTextField extends StatelessWidget {
   final TextEditingController controller;
   final String labelText;
-  final String assetIconPath;
+  final String svgIconPath;
+  final String prefixText;
+  final IconData icon;
   final TextInputType keyboardType;
   final bool obscureText;
   final String Function(String) onValidate;
   final Function(String) onChanged;
 
-  const AuthTextField({
-    this.labelText = '',
-    this.assetIconPath = '',
-    this.keyboardType = TextInputType.text,
-    this.obscureText = false,
-    this.controller,
-    this.onValidate,
-    this.onChanged,
-  });
+  const AuthTextField(
+      {this.labelText = '',
+      this.svgIconPath = '',
+      this.keyboardType = TextInputType.text,
+      this.obscureText = false,
+      this.controller,
+      this.onValidate,
+      this.prefixText,
+      this.onChanged,
+      this.icon});
```

Comment:
if u add a comma after icon we can save the structure:)

Reply:
fixed in next pr